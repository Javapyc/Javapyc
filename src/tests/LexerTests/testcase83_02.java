/* This is a tricky comment sequence * / note the space here *
/ note the newline // note the nested comment 
the end is coming soon... */
System.out.println*String+main/876789890({crazy][program});.!