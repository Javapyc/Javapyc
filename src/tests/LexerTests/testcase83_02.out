ReservedWord, System.out.println
Operator, *
ReservedWord, String
Operator, +
ReservedWord, main
Operator, /
Integer, 876789890
Delimiter, (
Delimiter, {
ID, crazy
Delimiter, ]
Delimiter, [
ID, program
Delimiter, }
Delimiter, )
Delimiter, ;
Delimiter, .
Operator, !
